### DISCUSSION TITLE
[PvP] Increase minimum player count required for battlegrounds on new season start

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#6705] [Suggestions]
Proposal to increase the amount of players required for battlegrounds to start once the new honor gear becomes available and the new arena season starts to the **blizzlike values** (full) for **a 2-3 weeks test period**.
(warsong requiring 20 players total, arathi 30, etc)

currently, battlegrounds can start with a less than full amount of players resulting in
- more battlegrounds
- potentially more imbalanced battlegrounds
- less blizzlike experience
- battlegrounds quicker affected by people leaving battlegrounds (1 of 10 is less relevant than 1 of 4)

the amount required was lowered in the level 19 bracket to allow battlegrounds to start at all. This results in a bad experience given battlegrounds are designed around their maximum capacity.

**Comment by tbcxd** [comment]
I support this idea, although it might have a negative impact, no bg at all is better than 2v2 wsg.
10 players min for wsg
16 Ab
18 Eots
AV 40
